modules/RTC/RTCUtils.js

```diff
@@ -398,27 +398,36 @@ function newGetConstraints(um = [], options = {}) {
             constraints.audio = {};
         }
 
-        // NOTE(brian): the new-style ('advanced' instead of 'optional')
-        // doesn't seem to carry through the googXXX constraints
-        // Changing back to 'optional' here (even with video using
-        // the 'advanced' style) allows them to be passed through
-        // but also requires the device id to capture to be set in optional
-        // as sourceId otherwise the constraints are considered malformed.
-        if (!constraints.audio.optional) {
-            constraints.audio.optional = [];
+        // Use the standard audio constraints on non-chromium browsers.
```

Comment:
Shouldn't we be more explicit here and use isChromiumBased / else?

Reply:
What about RN ?

Reply:
Those are not implemented anyway IIRC